### ISSUE TITLE
Enable auto closing the } bracket inside string templates

### ISSUE THREAD

**Issue by lidderupk** [Microsoft/vscode#28053]
This is an enhancement request. 

- VSCode Version: 1.12.2
- OS Version: macOS Sierra 10.12.3

Steps to Reproduce:

1. Type ```console.log(`this is ${`)```
2. Enable vscode to insert '}' and place cursor between {}.

```console.log(`this is ${|}`)```

**Comment by mjbvz** [comment]
This may depend on #34857

**Comment by mjbvz** [comment]
Auto-closing works in cases such as:

```
`a ${ a`
```

but not in cases like:

```
`a ${a`
`a ${`
```

The root cause is detailed #18120.

Our current behavior is consistent with sublime, but Atom has the described auto-closing behavior in the `a ${` case

**Comment by mjbvz** [comment]
Closing as duplicate of #18120
